### DISCUSSION TITLE
Is Vue2 not supported anymore?

### DISCUSSION THREAD

**Discussion by baranelitez** [meteorlxy/vue-showdown#30] [Q&A] [answered]
The 3.0.0 version is not working with Vue2, returning an `Uncaught TypeError: n.defineComponent is not a function` error. Is this intended or an oversight?

**Comment by meteorlxy** [comment, marked as answer]
In README:

![image](https://user-images.githubusercontent.com/18205362/115335868-a1843e00-a1d0-11eb-8cc6-9a30085cc823.png)

In docs:

![image](https://user-images.githubusercontent.com/18205362/115335887-a9dc7900-a1d0-11eb-9d64-5dffd1535eb0.png)
